src/role_squadheal.js

```diff
@@ -113,6 +114,6 @@ roles.squadheal.action = function(creep) {
   return true;
 };
 
-roles.squadheal.execute = function(creep) {
-  //  creep.log('Execute!!!');
-};
+//roles.squadheal.execute = function(creep) {
+//  creep.log('Execute!!!');
+//};
```

Comment:
Hm, not sure :-)

I think we should get rid of the `execute` function completely. This was the old system for executing creeps, if `action` returns false, the main loop will execute `execute`, this shouldn't happen anymore.